### PR TITLE
contrib/nload: new package (0.7.4)

### PR DESCRIPTION
**Pull request by firefly-cpp**
Package: http://www.roland-riegel.de/nload

### CODE REVIEW

**Comment by nekopsykose** [comment]
you're missing `automake` in hostmakedepends for the reconf to work (and maybe more stuff after)

**Comment by q66** [comment]
please test things *before* you open PRs for them

**Comment by firefly-cpp** [comment]
Thanks @nekopsykose.

File: contrib/nload/template.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
+pkgname = "nload"
+pkgver = "0.7.4"
+pkgrel = 0
+build_style = "gnu_configure"
+make_dir = "."
+hostmakedepends = ["autoconf", "automake"]
```
Comment (nekopsykose) [inline contrib/nload/template.py:6]: the autoconf is redundant, automake already pulls it

Comment (firefly-cpp) [reply contrib/nload/template.py]: fixed now

**Pull request merged**